README.md

```diff
@@ -168,8 +168,9 @@ Example: -r .jobscript.FastQC.*
 
 Please note that depending on your favorite shell, you may need quote your regex. 
 
-### Provide a file with several QBiC IDs
-In order to download datasets from several samples at once, you can provide a simple text file with multiple, line-separated, QBiC IDs and hand it to postman with the `-f` option.
+### Provide a manifest file with several QBiC IDs
+In order to download datasets from several samples at once, you can provide a manifest file consisting of multiple, line-separated, QBiC IDs.
+Hand it to postman with the `-f` option, a simple text file suffices here.
```

Comment:
```suggestion
Hand it to postman with the `-f` option.
```
I would keep it formal and maybe add an example file here. Otherwise, one might think that `.txt` files are expected.